src/integrationTest/groovy/wooga/gradle/paket/unity/PaketUnityFrameworksSpec.groovy

```diff
@@ -51,20 +51,47 @@ class PaketUnityFrameworksSpec extends IntegrationSpec {
         runTasksSuccessfully("paketInstall")
 
         then:
-        includeFrameworks.each {
+        includeFrameworks.every {
             def out1 = new File(projectDir, "Assets/${DefaultPaketUnityPluginExtension.DEFAULT_PAKET_DIRECTORY}/${dependency}/${it}")
             out1.exists()
         }
-        excludedFrameworks.each {
+
+        !excludedFrameworks.every {
```

Comment:
@marcolink I had to replace the `each` with ` `every` because the check `out1.exits()` never made it out of the closure.

Reply:
👍